### PR TITLE
exercises 26 09

### PR DESCRIPTION
**Pull request by AlexxSandra**
Write a program that takes a student's score as input and then prints their grade. Use the following grading scale:
90-100: A
80-89: B
70-79: C
60-69: D
Below 60: F
The following must be used:
1.	int
2.	char
3.	if / else if / else

### CODE REVIEW

File: src/main/java/alexadra/exercises/CalculateGradescore.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        } else if (score >= 80 && score <= 89) {
+            System.out.println("B");
+        } else if (score >= 70 && score <= 79) {
+            System.out.println("C");
+        } else if (score >= 60 && score <= 69) {
+            System.out.println("D");
+        } else {
+            System.out.println("F");
+        }
+    }
```
Comment (sabrina-graur) [inline src/main/java/alexadra/exercises/CalculateGradescore.java:22]: Please close scanner

File: src/main/java/alexadra/exercises/CalculateGradescore.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package alexadra.exercises;
+
+import java.util.Scanner;
+
+public class CalculateGradescore {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("enter student score: ");
+        int score = scanner.nextInt();
+
```
Comment (sabrina-graur) [inline src/main/java/alexadra/exercises/CalculateGradescore.java:10]: Remove space on line 10, please.

File: src/main/java/alexadra/exercises/CalculateGradescore.java (hunk 1 above)
Comment (AlexxSandra) [reply src/main/java/alexadra/exercises/CalculateGradescore.java:22]: Thank you, scanner was closed

File: src/main/java/alexadra/exercises/CalculateGradescore.java (hunk 2 above)
Comment (AlexxSandra) [reply src/main/java/alexadra/exercises/CalculateGradescore.java]: done

**Review by tudor-chirosca — CHANGES_REQUESTED** [review]
In the conditions it was mentioned that you must also use the char data type. Make the necessary changes, please. You can declare a variable of char data type, then change it value depending on the case.

File: src/main/java/alexadra/exercises/CalculateGradescore.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package alexadra.exercises;
+
+import java.util.Scanner;
+
+public class CalculateGradescore {
```
Comment (tudor-chirosca) [inline src/main/java/alexadra/exercises/CalculateGradescore.java:5]: Rename the class name. Check the Java naming conventions, please.

Comment (AlexxSandra) [reply src/main/java/alexadra/exercises/CalculateGradescore.java]: Thank you, changes where done

**Review by tudor-chirosca — APPROVED** [review]

**Review by sabrina-graur — APPROVED** [review]

**Pull request merged**